docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,190 @@
+  sudo apt-get install -y apt-transport-https ca-certificates curl
+  sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+  echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+  sudo apt-get update
+  sudo apt-get install -y kubectl
+  ```
+
+  > **Note:** The *kubectl* package is installed from Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository.
+
+  > **Note: All PMK cluster nodes would have kubectl installed once cluster is deployed.**
+
+- [Install or update Azure CLI (az) to version 2.25.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). 
+
+  Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+- [Helm](https://helm.sh/docs/intro/install/) version 3+ , to install the Azure Arc agents on the cluster.
+
+  ```shell
+  curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | sudo bash
+  ```
+
+- Create Azure service principal (SP)
+  To be able to complete the scenario and its related automation, Azure service principal assigned with the *Contributor* role is required.
+
+  For creating the service principal, first login to your Azure account.
+
+  ```shell
+  az login
+  ```
+
+  Login by opening the [Microsoft devicelogin page](https://microsoft.com/devicelogin) on browser and enter the code to authenticate.
+
+  An example output of successful authentication is shown below;
+
+  ![Screenshot of MSFT Device Login](./01.png)
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Screenshot of Creating Service Principal](./02.png)
+
+  > *Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a [specific Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).*
+
+  > **Note: Take a note of the *appId*, *password* and *tenant*, as that need to be set in an environment file later.**
+
+- Create an Azure resource group where you want the PMK cluster to show up.
+
+  ```shell
+  az group create -l <Azure Region> -n <resource group name>
+  ```
+
+  See below example of creating a resource group.
+  ![Screenshot of Creating Resource Group](./03.png)
+
+  You can verify it from the Azure portal.
+  ![Screenshot of Resource Group](./20.png)
```

Comment:
```suggestion
- Create an Azure resource group where you want the PMK cluster to show up.

  ```shell
  az group create -l <Azure Region> -n <resource group name>
  ```

  See below example of creating a resource group.
  ![Screenshot of Creating Resource Group](./03.png)

  You can verify it from the Azure portal.
  ![Screenshot of Resource Group](./20.png)
```

This should be part of the automation